src/stylesheets/components/_form-group.scss

```diff
@@ -0,0 +1,61 @@
+.form-group {
```

Comment:
@noor-ahmed I think we have this class in bootstrap already.
Is this about overriding? 🤔

Reply:
Yes.. for fulfilling the requirements we had to do this. Should I write another class?